core/error_macros.h

```diff
@@ -1,4 +1,4 @@
-/*************************************************************************/
+﻿/*************************************************************************/
```

Comment:
Did your IDE add a byte-order mark here?

Reply:
Yes. It's added a U+FEFF to the beginning of the line.

Reply:
I've removed it and updated the commit.